chord_metadata_service/patients/schemas.py

```diff
@@ -15,51 +17,38 @@
     description="Comorbid condition schema."
 )
 
+VITAL_STATUS_SCHEMA = tag_ids_and_describe({
+    "$schema": DRAFT_07,
+    "$id": "katsu:patients:individual",
+    "type": "object",
+    "properties": {
+        "status": enum_of(["UNKNOWN_STATUS", "ALIVE", "DECEASED"]),
+        "time_of_death": TIME_ELEMENT_SCHEMA,
+        "cause_of_death": ONTOLOGY_CLASS,
+        "survival_time_in_days": base_type(SCHEMA_TYPES.INTEGER)
+    },
+    "required": ["status"]
+}, VITAL_STATUS)
 
 INDIVIDUAL_SCHEMA = tag_ids_and_describe({
-    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$schema": DRAFT_07,
     "$id": "katsu:patients:individual",
     "type": "object",
     "properties": {
-        "id": {
-            "type": "string",
-            "description": "Unique researcher-specified identifier for the individual.",
-        },
-        "alternate_ids": {
-            "type": "array",
-            "items": {
-                "type": "string",
-            },
-            "description": "A list of alternative identifiers for the individual.",  # TODO: More specific
-        },
-        "date_of_birth": {
-            # TODO: This is a special ISO format... need UI for this
-            "type": "string",
-        },
-        "age": AGE_OR_AGE_RANGE,
-        "sex": {
-            "type": "string",
-            "enum": Sex.as_list(),
-            "description": "An individual's phenotypic sex.",
-        },
-        "karyotypic_sex": {
-            "type": "string",
-            "enum": KaryotypicSex.as_list(),
-            "description": "An individual's karyotypic sex.",
-        },
+        "id": base_type(SCHEMA_TYPES.STRING, description="Unique researcher-specified identifier for the individual."),
+        "alternate_ids": array_of(CURIE_SCHEMA, description="A list of alternative identifiers for the individual."),
+        "date_of_birth": DATE_TIME,
+        "time_at_last_encounter": TIME_ELEMENT_SCHEMA,
+        "vital_status": VITAL_STATUS_SCHEMA,
+        "sex": enum_of(Sex.as_list(), description="An individual's phenotypic sex."),
+        "karyotypic_sex": enum_of(KaryotypicSex.as_list(), description="An individual's karyotypic sex."),
+        "gender": enum_of(KaryotypicSex.as_list(), description="Self-identified gender"),
```

Comment:
hmmm... weird issue with phenopackets here. `KaryotypicSex`  (XX, XY, XXY etc) is clearly not the right way to represent "self-identified gender". Yet that's what it says to use in the [phenopackets 2.0.0 docs](https://phenopacket-schema.readthedocs.io/en/2.0.0/individual.html)! This is an error in the docs, which was corrected [here](https://github.com/phenopackets/phenopacket-schema/commit/7f1aac4d5b9875e35640488a8513220279f3a5e6). The [phenopackets spec](https://github.com/phenopackets/phenopacket-schema/blob/master/src/main/proto/phenopackets/schema/v2/core/individual.proto#L47-L48) says it should be OntologyClass rather than enum.

Reply:
good catch